Document/0x06a-Platform-Overview.md

```diff
@@ -131,27 +131,28 @@ A language.lproj folder is defined for each language that the application suppor
 
+
 #### App Structure on the iOS File System
 
-Since iOS 8, changes were made to the way an application is stored on the device. On versions before iOS 8, applications would be unpacked to a folder in the /var/mobile/applications/ folder. The application would be identified by its UUID (Universal Unique Identifier), a 128-bit number. This would be the name of the folder in which we will find the application itself. Since iOS 8 this has changed however, so we will see that the static bundle and the application data folders are now stored in different locations on the filesystem. These folders contain information that we will need to closely examine during application security assessments.
+iOS 8 introduced changes to the way an application is stored on a device. In the earlier versions of iOS applications would be unpacked to a folder in the /var/mobile/applications/ folder. An application would then be identified by its UUID (Universal Unique Identifier), a 128-bit number that will be the name of the folder where the application can be found. Since iOS 8 release static bundle and application data folders are now stored in different locations in the file system. These folders contain information that we must be closely examined during application security assessments.
 
 - `/var/mobile/Containers/Bundle/Application/[UUID]/Application.app` contains the previously mentioned application.app data and stores the static content as well as the ARM compiled binary of the application. The content of this folder will be used to validate the code signature.
-- `/var/mobile/Containers/Data/Application/[UUID]/Documents` contains all the data stored for the application itself. The creation of this data is initiated by the application’s end user.
-- `/var/mobile/Containers/Data/Application/[UUID]/Library` contains files necessary for the application e.g. caches, preferences, cookies, property list (plist) configuration files, etc.
-- `/var/mobile/Containers/Data/Application/[UUID]/Temp` contains temporary files which do not need persistence in between application launches.
+- `/var/mobile/Containers/Data/Application/[UUID]/Documents` contains all the data stored for the application itself. The creation of this data is initiated by the application end user.
+- `/var/mobile/Containers/Data/Application/[UUID]/Library` contains the files used by the application e.g. caches, preferences, cookies, property list (plist) configuration files, etc.
+- `/var/mobile/Containers/Data/Application/[UUID]/Temp` contains temporary files which are not needed between application launches.

```

Comment:
Please change "Temp" to "tmp" in the file path.

https://developer.apple.com/library/content/documentation/FileManagement/Conceptual/FileSystemProgrammingGuide/FileSystemOverview/FileSystemOverview.html